### PR TITLE
Unit test Validation and exception for AudioModel AudioParameter

### PR DESCRIPTION
**Pull request by julia-jpg**
Audio Validations and Exceptions

### CODE REVIEW

File: src/main/java/org/fundacionjala/converter/model/command/multimedia/AudioModel.java [hunk 1]
```diff
@@ -85,26 +79,23 @@ private void addToList(final List<String> list, final String condition, final St
...
     }
     private List<String> extractMetadata(final AudioParameter audioParameter, final String suffix) {
-        String input = audioParameter.getOutputFile() + audioParameter.getName() + suffix + audioParameter.getFormat();
-        String output = audioParameter.getName() + suffix + audioParameter.getFormat().substring(1) + META_SUFFIX;
+        String fileConvertPath = ConfigPath.getConvertedFilesPath();
+        String input = fileConvertPath + audioParameter.getMd5() + suffix + audioParameter.getFormat();
+        String output = "";
         MetadataParameter metadataParameter = new MetadataParameter(input, JSON, VERBOSE, output, VOID);
         MetadataModel metadataModel = new MetadataModel();
         return metadataModel.createCommand(metadataParameter).get(0);
```
Comment (roht0515) [inline src/main/java/org/fundacionjala/converter/model/command/multimedia/AudioModel.java:105]: return metadataModel.createCommand(metadataParameter).get(0);
why do you use "0"?

Comment (MirkoFer-622) [reply src/main/java/org/fundacionjala/converter/model/command/multimedia/AudioModel.java:105]: If you really need to use "0", avoid to use magic numbers

**Comment by MirkoFer-622** [comment]
Please, solve conflicts to merge it

File: src/main/java/org/fundacionjala/converter/model/configPath/ConfigHandler.java [hunk 2]
```diff
@@ -11,7 +11,7 @@
     private ConfigHandler() {
         try {
             this.properties = new Properties();
-            if ((System.getProperty("os.name").toLowerCase()).contains("win")) {
+            if ((System.getProperty("os.name").toLowerCase()).contains("Win")) {
```
Comment (samuelloza) [inline src/main/java/org/fundacionjala/converter/model/configPath/ConfigHandler.java]: Please, revert this line

**Review by Laura4lilavati — APPROVED** [review]

**Review by samuelloza — APPROVED** [review]

**Pull request merged**